### DISCUSSION TITLE
Add custom modes to the right-click G icon

### DISCUSSION THREAD

**Discussion by ggjiza** [seerge/g-helper#1444] [Ideas]
![Screenshot 2023-10-07 032537](https://github.com/seerge/g-helper/assets/133283942/96597fb0-09e1-468b-9d35-9b76049c3d70)
Currently, if you right-click Ghelper on the taskbar you see modes & GPU modes, yet that doesn't include the custom modes created.
Having the most recent custom mode/all of them would be more efficient to choose rather than going through the "Fans+Power" or toggling through all modes to reach the one wanted.

**Comment by seerge** [comment]
@ggjiza hello, it doesn't include them but it's by design. That context menu is quite difficult to manipulate, and I prefer to keep UI simple.

You can always switch to next / previous mode with ``Fn + F5`` / ``Fn + Shift + F5`` :)

**Reply by ggjiza** [reply]
I see, thanks. 
I'll probably optimize the stock modes so I can rely on those.
I remember seeing power limits for the different modes and devices somewhere on the page but I can't find it now, any chance you might know where to look?

**Reply by seerge** [reply]
Power limits are under Fans+Power -> CPU section

**Reply by ggjiza** [reply]
Oh no sorry, I meant the default values set from Asus.
the power limit bar Is always sitting on 125W by default (in my case my device is M16 12900H) for all modes, granted "apply power limits" is unchecked. I'm also assuming that the scale (max is 145W) is just a constant; doesn't reflect the device's actual max.
I want to know the default values that are set when "apply power limits" is unchecked.

**Reply by seerge** [reply]
@ggjiza you can't read them anyhow directly. The only reliable way is to run something like cinebench in conjunction with hwinfo and literally monitor how much watts does your CPU consume when unleashed
